src/offline/component.js

```diff
@@ -205,6 +211,41 @@ exports.Controller_ = class {
      */
     this.displayAlertAbortDownload = false;
 
+    /**
+     * Offline mask minimum margin in pixels.
+     * @type {number}
+     * @export
+     */
+    this.maskMargin = this.maskMargin || 100;
+
+    /**
+     * Minimum zoom where offline is enable.
+     * @type {number}
+     * @export
+     */
+    this.minZoom = this.minZoom || 10;
+
+    /**
+     * Maximum zoom where offline is enable.
+     * @type {number}
+     * @export
+     */
+    this.maxZoom = this.maxZoom || 15;
```

Comment:
At this point, the default value will be always used. Because the values from the bindings are analyzed only from the "onInit" methode.
So perhaps you can write here `this.maxZoom = 15;` (and that's all.) And perhaps it's better to write `this.maxZoom || 15;` in the onInit.

Reply:
Same for the other bindings